test/check-loopback

```diff
@@ -42,7 +54,8 @@ class TestLoopback(MachineCase):
         b.wait_visible('#login-fatal')
         self.assertIn("The server refused to authenticate 'admin' using password authentication, and no other supported authentication methods are available.", b.text('#login-fatal'));
 
-        self.allow_journal_messages("cockpit-polkit: couldn't respond to polkit daemon: GDBus\\.Error:org\\.freedesktop\\.PolicyKit1\\.Error\\.Failed: No session for cookie")
+        self.allow_journal_messages("cockpit-polkit: couldn't respond to polkit daemon: GDBus\\.Error:org\\.freedesktop\\.PolicyKit1\\.Error\\.Failed: No session for cookie",
+                                    ".*server offered unsupported authentication methods: public-key.*")
```

Comment:
>
> Would it make sense to remove authentication-not-supported and
> no-forwarding completely (by returning authentication-failed instead)
> and just use the more explicit auth-methods-results member?

I'm going to try to clean this up when I get around to #3269. We'll 
remove no-forwarding and only return authentication-not-supported when 
truly nothing is supported instead of not tried. login.html will need to 
check auth-methods-results for displaying it's error. Sound Good?